app/assets/v2/js/activity.js

```diff
@@ -17,6 +17,23 @@ $(document).ready(function() {
     });
   }
 
+  $(document).on('click', '.click_here_to_join_video', function(e) {
+    e.preventDefault();
+    $(this).text('');
+    var roomName = $(this).data('roomname');
+    var jitsi_domain = 'meet.jit.si';
+    var jitsi_options = {
+      roomName: roomName,
+      parentNode: document.querySelector('#' + roomName),
+      welcomePageEnabled: true,
+      userInfo: {
+        displayName: document.contxt.github_handle ? document.contxt.github_handle : 'Anonymous',
+        email: document.contxt.email ? document.contxt.email : null
+      }
+    };
+    var jitsi_api = new JitsiMeetExternalAPI(jitsi_domain, jitsi_options);
```

Comment:
Do we need this in a variable ?
Also var -> let/const :P

Reply:
yep need to be in a variable (const works). As it need to run dispose when ends the call and others related methods.

Reply:
OK

Reply:
done

Reply:
still showing me 'var ' for me

Reply:
stupid github, seems I can see the changes just here https://github.com/gitcoinco/web/pull/6250/commits/3a9290216b247d34a3c8e73ae4c3931d025d6125